deployment.yaml.erb

```diff
@@ -3,7 +3,6 @@
 <%
 def set_node_template_vars(values)
   @Eth_Etherbase         = values["geth"]["Eth_Etherbase"]
-  @Eth_MinerThreads      = values["geth"]["Eth_MinerThreads"]
```

Comment:
I agree this can be dropped, as it seems to have no effect on the config file. however when we drop it could you also drop it from the other places? README.md, kuberneteth, kuberneteth.yaml, scripts/generate_nodes.sh come to my mind